libcontainer/cgroups/utils_test.go

```diff
@@ -661,3 +663,29 @@ func TestConvertBlkIOToIOWeightValue(t *testing.T) {
 		}
 	}
 }
+
+// TestRemovePathEROFS is to test remove a non-existent dir in a ro mount point.
+// The similar issue example: https://github.com/opencontainers/runc/issues/4518
+func TestRemovePathEROFS(t *testing.T) {
```

Comment:
```suggestion
func TestRemovePathReadOnly(t *testing.T) {
```

(I thought this test involved with the EROFS file system)